Makefile-man.am

```diff
@@ -26,10 +26,11 @@ ostree-admin-status.1 ostree-admin-set-origin.1 ostree-admin-switch.1	\
 ostree-admin-undeploy.1 ostree-admin-upgrade.1 ostree-admin-unlock.1	\
 ostree-admin.1 ostree-cat.1 ostree-checkout.1 ostree-checksum.1		\
 ostree-commit.1 ostree-export.1 ostree-gpg-sign.1 ostree-config.1	\
-ostree-diff.1 ostree-fsck.1 ostree-init.1 ostree-log.1 ostree-ls.1	\
-ostree-prune.1 ostree-pull-local.1 ostree-pull.1 ostree-refs.1		\
-ostree-remote.1 ostree-reset.1 ostree-rev-parse.1 ostree-show.1		\
-ostree-summary.1 ostree-static-delta.1
+ostree-diff.1 ostree-find-remotes.1 ostree-fsck.1 ostree-init.1 \
```

Comment:
Should this one be conditional on `ENABLE_EXPERIMENTAL_API` ?

Reply:
Yeah I suppose so